src/lib.rs

```diff
@@ -628,27 +635,54 @@ pub use async_graphql_derive::Union;
 /// # Examples
 ///
 /// ```ignore
-/// use async_graphql::*;
+/// use async_graphql::prelude::*;
 ///
-/// #[Object]
+/// #[GqlObject]
 /// struct Event {
 ///     value: i32,
 /// }
 ///
 /// struct SubscriptionRoot;
 ///
-/// #[Subscription]
+/// #[GqlSubscription]
 /// impl SubscriptionRoot {
 ///     async fn value(&self, event: &Event, condition: i32) -> bool {
 ///         // Push when value is greater than condition
 ///         event.value > condition
 ///     }
 /// }
 /// ```
-pub use async_graphql_derive::Subscription;
+pub use async_graphql_derive::GqlSubscription;
 
 /// Define a DataSource
-pub use async_graphql_derive::DataSource;
+pub use async_graphql_derive::GqlDataSource;
 
 /// Define a Scalar
-pub use async_graphql_derive::Scalar;
+pub use async_graphql_derive::GqlScalar;
+
+mod deprecated {
+    pub use super::GqlContext as Context;
+    pub use super::GqlContextBase as ContextBase;
+    pub use super::GqlData as Data;
+    pub use super::GqlDataSource as DataSource;
+    pub use super::GqlEnum as Enum;
+    pub use super::GqlError as Error;
+    pub use super::GqlFieldResult as FieldResult;
+    pub use super::GqlID as ID;
+    pub use super::GqlInputObject as InputObject;
+    pub use super::GqlInputValueResult as InputValueResult;
+    pub use super::GqlInterface as Interface;
+    pub use super::GqlObject as Object;
+    pub use super::GqlQueryBuilder as QueryBuilder;
+    pub use super::GqlResult as Result;
+    pub use super::GqlScalar as Scalar;
+    pub use super::GqlSchema as Schema;
+    pub use super::GqlSimpleObject as SimpleObject;
+    pub use super::GqlSubscription as Subscription;
+    pub use super::GqlUnion as Union;
+    pub use super::GqlValue as Value;
+    pub use super::GqlVariables as Variables;
+}
+
+#[doc(hidden)]
+pub use deprecated::*;
```

Comment:
Previous exports names are still available here, the whole project build and pass tests with this module disabled